src/pal/src/misc/miscpalapi.cpp

```diff
@@ -298,3 +301,153 @@ PAL_Random(
     PERF_EXIT(PAL_Random);
     return bRet;
 }
+
+
+// Support for EnsureOpenSslInitialized
+
+static const char * const libcryptoName = "libcrypto" PAL_SHLIB_SUFFIX;
```

Comment:
Thanks, Geoff. Do I need to hardcode a particular version in the name, or is it possible to load whatever version is available on the system?